packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -225,6 +225,7 @@ const UsersProfileFn = ({terms, slug, classes}: {
       <div className={classNames("page", "users-profile", classes.profilePage)}>
         <HeadTags
           description={metaDescription}
+          noIndex={!user.postCount || !user.commentCount || user.karma <= 0}
```

Comment:
If a user has comments but not posts (or vise versa) it probably makes sense to index them?

Maybe instead: `(!user.postCount && !user.commentCount) || user.karma<=0`

Reply:
Yeah, I think this should change. The thing Jim suggested seems better to me. I think it would be quite bad to not index users with only comments or only posts.

Reply:
Oh, yeah, that makes sense.